src/ImageSharp/Color/Color.WernerPalette.cs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+
+namespace SixLabors.ImageSharp
+{
+    /// <content>
+    /// Contains the definition of <see cref="WernerPalette"/>.
+    /// </content>
+    public partial struct Color
+    {
+        private static readonly Lazy<Color[]> WernerPaletteLazy = new Lazy<Color[]>(CreateWernerPalette, true);
+
+        /// <summary>
+        /// Gets a collection of colors as defined in the original second edition of Werner’s Nomenclature of Colours 1821.
+        /// The hex codes were collected and defined by Nicholas Rougeux <see href="https://www.c82.net/werner"/>.
+        /// </summary>
+        public static ReadOnlyMemory<Color> WernerPalette => WernerPaletteLazy.Value;
+
+        private static Color[] CreateWernerPalette() => new[]
```

Comment:
Should we cast from `NamedColors` or remove the separate implementation entirely?